specs/0119-transactional-emails.md

```diff
@@ -0,0 +1,428 @@
+
+If you have a question or issue related to a particular string, you can click the “REQUEST CONTEXT or REPORT ISSUE” button from the translation view in Pontoon. That will create a comment you can fill in with the details and it will notify the project manager. You can also find the project manager’s contact information in their profile by clicking on their name in the field at the top of the project page.
+
+We hope this gives you a better understanding of how localization works at Mozilla. 
+
+If you have any questions, reach out any time to the Mozilla localization team on [Matrix chat](https://matrix.to/#/#l10n-community:mozilla.org) or [Discourse](https://discourse.mozilla.org/c/l10n/547).
+
+Thanks,  
+Pontoon Team
+
+#### Frequency
+
+Once
+
+### Onboarding: Week 1 - more localization resources
+
+#### Trigger
+
+When both conditions are met: 
+
+1. At least 1 week after account creation  
+2. 5 or more translation suggestions submitted
+
+### Template
+
+Subject: More resources for contributors  
+Body:  
+Hello,
+
+Thank you so much for contributing translations to Mozilla localization! This is our final introductory email - so from here on out you won’t find us in your inbox (excluding essential account related communications) unless you choose to sign up for the future email updates described below.
+
+Since you’ve had a chance to make some translation suggestions, we’d like to provide you with some additional resources that might be useful as you continue in your journey as a member of our localization community.
+
+**Additional documentation:**  
+For in-depth documentation on Mozilla localization, see our [Documentation for Mozilla localizers](https://mozilla-l10n.github.io/localizer-documentation/index.html) which includes in depth explanations on topics such as:
+
+* [Community roles](https://mozilla-l10n.github.io/localizer-documentation/community/l10n_community_roles.html)  
+* Specific [Mozilla product](https://mozilla-l10n.github.io/localizer-documentation/products/index.html) localization (such as Firefox, Firefox for Android, etc.)  
+* Tools such [Pontoon](https://mozilla-l10n.github.io/localizer-documentation/tools/pontoon/index.html) and [Fluent](https://mozilla-l10n.github.io/localizer-documentation/tools/fluent/index.html) (the advanced translation format used by Mozilla)
+
+**Pontoon Add-on:**   
+You can use the Pontoon Add-on for [Firefox](https://mozilla-l10n.github.io/localizer-documentation/index.html) or [Chrome](https://chromewebstore.google.com/detail/pontoon-add-on/gnbfbnpjncpghhjmmhklfhcglbopagbb) to be instantly notified of any Pontoon notifications right in your browser, as well as some helpful tools to help with proofreading, bug reporting, and fixing. 
+
+**Mozilla community participation guidelines:**  
```

Comment:
This is already mentioned in the welcome email.

Reply:
Yes, but still worthwhile to include in this since it's 1: important and 2: this is sort of a final collection of useful resources.